### PR TITLE
Remove stage idx

### PR DESCRIPTION
**Pull request by tokatoka**
This stage_idx is not used anywhere

### CODE REVIEW

**Comment by domenukk** [comment]
@andreafioraldi yes/no?
@tokatoka oops conflicts :D

File: libafl_targets/src/cmplog.rs [hunk 1]
```diff
@@ -28,7 +28,7 @@ pub const CMPLOG_MAP_RTN_H: usize = (CMPLOG_MAP_H * core::mem::size_of::<CmpLogI
 
 /// `CmpLog` instruction kind
 pub const CMPLOG_KIND_INS: u8 = 0;
-/// `CmpLog` return kind
+/// `CmpLog` routine kind
```
Comment (domenukk) [inline libafl_targets/src/cmplog.rs:31]: This doc fix we should take in any case

**Comment by tokatoka** [comment]
Close as it could be useful in the future

File: libafl_targets/src/cmplog.rs (hunk 1 above)
Comment (tokatoka) [reply libafl_targets/src/cmplog.rs:31]: OK

**Pull request closed without merging**